media/css/mozorg/contribute/contribute-2015.less

```diff
@@ -1472,12 +1472,21 @@ textarea:focus {
     display: none;
 }
 
+.mozillians-profile.friends {
```

Comment:
You could get away with just the `.friends` selector here I think? And just `.oneandone` below?